Sources/Misc/DangerousSettings.swift

```diff
@@ -13,14 +13,11 @@ import Foundation
  */
 @objc(RCDangerousSettings) public final class DangerousSettings: NSObject {
 
-    /// Dangerous settings not exposed outside of the SDK.
-    internal struct InternalSettings {
+    internal struct Internal: InternalDangerousSettingsType {
```

Comment:
Changed this from `DangerousSettings.InternalSettings` to `DangerousSettings.Internal`.